apps/api/src/server/applications/application/documents/document.controller.js

```diff
@@ -53,6 +54,33 @@ export const provideDocumentUploadURLs = async ({ params, body }, response) => {
 	});
 };
 
+/**
+ *
+ * @type {import('express').RequestHandler<any, any, { blobStorageHost: string, blobStorageContainer: string, documents: { documentName: string, blobStoreUrl: string }[] } | any, any>}
+ */
+export const provideDocumentVersionUploadURL = async ({ params, body }, response) => {
+	const documentsToUpload = body;
```

Comment:
Is this right that we're supporting providing multiple documents? I think new versions will only be uploaded one at a time